src/redux/books/books.js

```diff
@@ -0,0 +1,31 @@
+const ADD_BOOK = 'book-store/books/ADD';
+const REMOVE_BOOK = 'book-store/books/REMOVE';
+
+const initialState = {
+  books: [],
+};
+
+export default function reducer(state = initialState, action) {
+  switch (action.type) {
+    case ADD_BOOK:
+      return {
+        ...state,
+        books: [...state.books, action.payload],
+      };
+    case REMOVE_BOOK:
+      return {
+        ...state,
+        books: [],
+      };
```

Comment:
- [ ] Good job setting up the reducers. However, this action causes the list of books to be completely removed, instead of removing the desired book.

- You can easily filter out the unwanted book (which you can pass the `id` of, in the `payload`) by using the [Array `filter` method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter).